english/macgyver/toss-lighter/Stand/Stand_like_goon.rm

```diff
@@ -0,0 +1 @@
+As you stand there and drool, the smell becomes more pungent
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.